packages/qwik-city/runtime/src/api.md

```diff
@@ -167,6 +167,8 @@ export interface DocumentLink {
     type?: string;
 }
 
+type MetaNames =  "application-name" | "author" | "description" | "generator" | "keywords" | "referrer" | "theme-color" | "color-scheme" | "viewport" | "creator" | "googlebot" | "publisher" | "robots";

```

Comment:
I don't think this is an exhaustive list Where are these defined?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name

I don't think there is an exhaustive list, so such a change would make this breaking to our users.